src/wandbot/apps/zendesk/extract_by_type.py

```diff
@@ -0,0 +1,13 @@
+def discourseExt(description) -> str:
+    question = description.lower().replace('\n', ' ').replace('\r', '')
+    question = question.replace('[discourse post]','')
+    question = question[:4095]
```

Comment:
what are the risks of curtailing here? often I see users post their question after a very long stack trace. In that case the user query might be removed here. I think this needs a better way to handle longer strings